Atomic/help.py

```diff
@@ -0,0 +1,41 @@
+from . import Atomic
+import subprocess
+from pydoc import pager
+from os import environ
+from . import mount
+import sys
+
+class AtomicHelp(Atomic):
+    def help(self):
+        """
+        Displays the image/containe help file in a pager
+        :return: None
+        """
+        # Set the pager to less -R
+        dm = mount.DockerMount("/tmp")
+        enc=sys.getdefaultencoding()
+        environ['PAGER'] = '/usr/bin/less -R'
+        docker_id = self.get_input_id(self.args.docker_object)
+        self.inspect = self._inspect_image(docker_id)
+        help_id = docker_id
+        if self.inspect is None: # docker_id is a container
+            self.inspect = self._inspect_container(docker_id)
+            help_id = dm._clone(docker_id, image_only=True)
+        # Check if the help path has been overridden
+        alt_help_path = self._get_labels().get('HELP')
+        help_file = alt_help_path if alt_help_path is not None else '/image_help.1'
+        cmd1 = ['docker', 'run', '--rm', help_id, 'cat', help_file]
+        cmd2 = ['groff', '-man', '-Tascii']
```

Comment:
What happens if we don't have cat?  Couldn't we just atomic mount the image and then use local tools like groff, rather then docker run.